### ISSUE TITLE
Comic tagging makes files inaccessible

### ISSUE THREAD

**Issue by ysarkain** [mylar3/mylar3#1480]
**Describe the bug**
When I meta-tag comics using Mylar, the resulting cbz file has permissions set to 600, preventing Komga and Cover from opening or reading the files.  I am able to fix the issue by running chmod -R 766 on the comics mount point after each batch, but it's a hassle and I'm trying to figure out why the meta-tagger is setting permissions this way.

**CarePackage**
CarePackage was too big at 48MB, however no errors appear in the logs.

**Environment (please complete the following information):**
 - OS: Unraid (docker)
 - Last updated 1/16/24
 - Branch / commit: Version [v0.7.8]
 - Python Version : 3.11.6
 - Language : en_US.UTF-8

**Additional information**
Unraid sets owner:group as nobody:users, 
I copy files over from a Windows PC using SMB and verified that ownership is unaffected. Permissions are 766 when I copy them in.
Renaming has no effect on permissions or ownership, but meta-tagging sets permissions to 600 and does not change ownership.
I tried changing the umask in the docker settings from 022 to 002 and 011, with no effect, meta-tagger sets permissions on tagged files to 600, so I have to chmod to regain read/write access to the file.

**Comment by falo2k** [comment]
Have a look at the [Perms] section under config.ini.  You should be able to get it to set correct permissions there.

I have also found that Unraid can be a bit iffy with setting perms properly following samba copies, depending on the share settings.  As a bonus, you may want to take a look at this thread (though the config these days can be directly added into the Samba settings page for "extra configuration") https://forums.unraid.net/topic/101956-smb-writes-wrong-permissions-solved/

Bonus 2: In case you weren't aware, you can do the equivalent of using the New Permissions tool at a shell by running:
`newperms /absolute/path/to/update`
This will ensure everything below that path has appropriate perms, ownership, etc.  but don't run it against appdata.

**Comment by ysarkain** [comment]
Yeah, my workaround is a simple alias: [code]alias fixcomics='chmod -R 776 /mnt/user/archive/comics'[/code].  

My solution for SMB permission issues was to set 'force user = nobody' in the samba.cfg for the 'archive' mount, before I did that meta-tagging worked only about 15% of the time, since the mylar user wouldn't have appropriate rights to the files.

I saw the section for perms in the config.ini and then found the little checkbox in settings to enforce permissions, set that to 777 for folders and 776 for files, and ownership abc:abc and everything seems to be working smoothly now.

Thanks!
